XRTK-Core/Packages/com.xrtk.core/Services/InputSimulationSystem/SimulatedHandData.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Services.InputSimulationSystem
```

Comment:
Can we change the namespace to 
```
XRTK.Services.InputSystem.Simulation
```
Makes more sense that way, as the input system is required for simulation, it's not a replacement